dot/dot.go

```diff
@@ -30,15 +31,17 @@ import (
 type Dot struct {
 	Services  *services.ServiceRegistry // Registry of all core services
 	Rpc       *rpc.HttpServer           // HTTP instance for RPC server
+	Manager   *core.Service             // service manager
```

Comment:
This should be included as a service, not passed in as a param. You just append it to the `srvcs` list and then `Dot` will call its `Start` function

Reply:
yeah the issue is that in the configcmd test `TestGenesisStateLoading` I'm not sure how to check that the storage trie has been properly loaded into the service after calling makeNode, since a service only has Start and Stop

Reply:
Perhaps `services.Get()` is what you're looking for?

Reply:
nice, changed the test and removed this from dot